draft-js-mention-plugin/src/MentionSuggestions/index.js

```diff
@@ -58,7 +58,9 @@ export default class MentionSuggestions extends Component {
           focusedOptionIndex: size - 1,
         });
       }
-
+      if (!this.props.store.getAllSearches().has(this.activeOffsetKey)) {
```

Comment:
yes, it won't work for mention trigger more than 1 char. but at least the code doesn't break now..I think we may either add docs to tell user that mention trigger only can be set as single character or     add support for mulit-character mention trigger.